pages/eco_news_page.py

```
@@ -0,0 +1,75 @@
+
+
+    def click_ads_filter(self):
+        news_filter_button = self.driver.find_element(By.XPATH, ads_tag_button)
+        news_filter_button.click()
+        self.driver.implicitly_wait(5)
+        self.driver.refresh()
+
+    def get_tags_of_first_and_second_news(self):
+        #TODO: add "try" for case when news doesn't exist
+
+        first_element = self.driver.find_element(By.XPATH, first_news_tags_list)
```

Ruff (0.12.2)

`By` may be undefined, or defined from star imports

(F405)